lib/main_publiccloud.pm

```diff
@@ -13,19 +13,12 @@ use version_utils;
 use main_common qw(loadtest);
 use testapi qw(check_var get_var);
 use Utils::Architectures qw(is_aarch64);
+use main_containers qw(load_3rd_party_image_test);
 
 our @EXPORT = qw(
   load_publiccloud_tests
 );
 
-sub load_3rd_party_image_test {
-    my ($runtime) = @_;
-    my $args = OpenQA::Test::RunArgs->new();
```

Comment:
Why do we generate a new RunArgs instance here? It might work (for now) but this is wrong! Every test met get the same instance of RunArgs, otherwise it's purpose is defied.

I know this is not this PR, but AFAICS this is wrong.

Reply:
Yeah, I was probably too rushed merging this... sorry. Wanted to get a quick fix and made it worse :) 
Thanks Pavel for taking care.

Reply:
I agree. I created [poo#107230](https://progress.opensuse.org/issues/107230)